instrumentation/jetty/jetty-8.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/jetty/v8_0/Jetty8AttributesExtractor.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.jetty.v8_0;
+
+import io.opentelemetry.instrumentation.api.instrumenter.http.HttpAttributesExtractor;
+import io.opentelemetry.semconv.trace.attributes.SemanticAttributes;
+import java.net.URI;
+import java.net.URISyntaxException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+public class Jetty8AttributesExtractor
+    extends HttpAttributesExtractor<HttpServletRequest, HttpServletResponse> {
+
+  @Override
+  protected @Nullable String method(HttpServletRequest httpServletRequest) {
+    return httpServletRequest.getMethod();
+  }
+
+  @Override
+  protected @Nullable String url(HttpServletRequest httpServletRequest) {
+    URI uri = URI.create(httpServletRequest.getRequestURL().toString());
```

Comment:
I believe you don't really need to construct this `URI`, just get everything you need from `HttpServletRequest` like in `ServletHttpServerTracer.url()`